assets/src/block-validation/style.css

```diff
@@ -1,10 +1,10 @@
-.components-button[aria-label="AMP for WordPress"],
-.components-button[aria-label="AMP for WordPress"]:hover,
-.components-button[aria-label="AMP for WordPress"]:focus {
+.components-button[aria-label="AMP"],
+.components-button[aria-label="AMP"]:hover,
+.components-button[aria-label="AMP"]:focus {
 	color: transparent;
```

Comment:
I don't see what this `transparent` color was being used for. I removed it in 7455c5726500655bc68342d9a4141aa1a3d8e7c4.